src/pages/messages/components/MessageMenu.tsx

```diff
@@ -0,0 +1,79 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import { IconButton, ListItemIcon, ListItemText, Menu, MenuItem, Typography } from '@mui/material';
+import { MouseEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useDeleteTicketMessage } from '../../../utils/hooks/useDeleteTicketMessage';
+
+interface MessageMenuProps {
+  canDeleteMessage: boolean;
+  ticketId: string;
+  messageIdentifier: string;
+  refetchData?: () => void;
+}
+
+export const MessageMenu = ({ ticketId, messageIdentifier, refetchData, canDeleteMessage }: MessageMenuProps) => {
+  const { t } = useTranslation();
+  const [showConfirmDialog, setShowConfirmDialog] = useState(false);
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const deleteTicketMessageMutation = useDeleteTicketMessage(ticketId, refetchData);
+
+  const handleClickMenuAnchor = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  return (
+    <>
+      <section>
```

Comment:
Kunne fjerna en tag, ved å legge `<ConfirmDialog>` inni `<section>`, men går for det samme. Your call
```suggestion
    <section>
```